header.php

```diff
@@ -46,7 +46,24 @@
         </ul>
       </nav>
     </div>
+   </div>
 
 <!--  <img src="http://Setupshop/wp-content/uploads/2016/09/SA.png" alt="sa-flags">  -->
 
-<?php dynamic_sidebar( 'Filters' ); ?>
\ No newline at end of file
+<!--<?php dynamic_sidebar( 'Filters' ); ?>-->
+
+<div class="container">
+  <section>
+  	<article id="desc">
```

Comment:
Try and minimise the use of IDs. It will only make the risk of duplicates higher. If you're not referring to it in CSS or JS theres no need to give it an ID or Class